CHANGELOG.md

```diff
@@ -1,54 +1,11 @@
 # Changelog
 
-All notable changes to Kubernetes and Helm resources for WSO2 IAM version `5.11.x` in each resource release, will be documented in this file.
+All notable changes to Kubernetes and Helm resources for WSO2 IAM version `6.0.x` in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v5.11.0.5] - 2021.11.25
+## [v6.0.0.1] - 2022-08-17
 
 ### Added
 
-- Integrate Kubernetes Startup Probe to Identity Server workload (refer [issue](https://github.com/wso2/kubernetes-is/issues/230))
-- Use WSO2 Identity Server Health Check API for Kubernetes Readiness Probes (refer [issue](https://github.com/wso2/kubernetes-is/issues/252))
-
-### Changed
-
-- Upgrade the API version of Kubernetes Ingress resource (refer [issue](https://github.com/wso2/kubernetes-is/issues/312))
```

Comment:
Also shouldnt we add about the user store change and the jdk support cc: @piraveena

Reply:
+1 to add.